Source/SA/RFmxDPD/RFmxDPD.cs

```diff
@@ -0,0 +1,202 @@
+                    PreDpdCFREnabled = RFmxSpecAnMXDpdPreDpdCfrEnabled.False,
+                    ApplyDpdCFREnabled = RFmxSpecAnMXDpdApplyDpdCfrEnabled.False
+                };
+            }
+
+        }
+        public struct LookupTableConfiguration
+        {
+            public RFmxSpecAnMXDpdLookupTableThresholdType ThresholdType;
+            public RFmxSpecAnMXDpdLookupTableThresholdEnabled ThresholdEnabled;
+            public double ThresholdLevel;
+            public double LookupTableStepSize;
+            public RFmxSpecAnMXDpdApplyDpdLookupTableCorrectionType LookupTableCorrectionType;
+            public RFmxSpecAnMXDpdLookupTableType LookupTableType;
+
+            public static LookupTableConfiguration GetDefault()
+            {
+                return new LookupTableConfiguration
+                {
+                    LookupTableType = RFmxSpecAnMXDpdLookupTableType.Linear,
+                    ThresholdType = RFmxSpecAnMXDpdLookupTableThresholdType.Relative,
+                    ThresholdEnabled = RFmxSpecAnMXDpdLookupTableThresholdEnabled.True,
+                    ThresholdLevel = -20,
+                    LookupTableStepSize = 0.1,
+                    LookupTableCorrectionType = RFmxSpecAnMXDpdApplyDpdLookupTableCorrectionType.MagnitudeAndPhase
+
+                };
+            }
+        }
+
+        public struct MemoryPolynomialConfiguration
+        {
+            public RFmxSpecAnMXDpdApplyDpdMemoryModelCorrectionType MemoryModelCorrectionType;
+            public RFmxSpecAnMXDpdIterativeDpdEnabled IterativeDpdEnabled;
+            public int NumberOfIteration;
+            public int MemoryPolynomialOrder, MemoryPolynomialDepth, MemoryPolynomialLeadOrder, MemoryPolynomialLagOrder,
+            MemoryPolynomialLeadMemoryDepth, MemoryPolynomialLagMemoryDepth, MemoryPolynomialMaxLead, MemoryPolynomialMaxLag;
+            public static MemoryPolynomialConfiguration GetDefault()
+            {
+                return new MemoryPolynomialConfiguration
+                {
+                    MemoryModelCorrectionType = RFmxSpecAnMXDpdApplyDpdMemoryModelCorrectionType.MagnitudeAndPhase,
+                    IterativeDpdEnabled = RFmxSpecAnMXDpdIterativeDpdEnabled.False,
+                    NumberOfIteration = 3,
+                    MemoryPolynomialDepth = 2,
+                    MemoryPolynomialOrder = 3,
+                    MemoryPolynomialLeadOrder = 2,
+                    MemoryPolynomialLagOrder = 2,
+                    MemoryPolynomialLeadMemoryDepth = 2,
+                    MemoryPolynomialLagMemoryDepth = 2,
+                    MemoryPolynomialMaxLead = 2,
+                    MemoryPolynomialMaxLag = 2
+                };
+            }
+        }
+
+        public struct LookupTableResults
+        {
+            public SG.Waveform PostDPDWaveform;
+            public float[] LookupTableInputPowers;
+            public ComplexSingle[] LookupTableComplexGains;
+            public double PostDPDPAPR;
```

Comment:
```suggestion
            public double PostDPDPapr_dB;
```
Two acronyms one after the other is hard to read